doc/manuals/user/known_limitations.md

```diff
@@ -62,21 +50,6 @@ Required" response. This is due to the way the underlying HTTP library
 mailing
 list](http://lists.gnu.org/archive/html/libmicrohttpd/2014-01/msg00063.html).
 
-## Entity fields length limitation
```

Comment:
These two sections are related with NGSIv1 limitations so I have taken the opportunity to remove them (documentation about deprecated functionality is removed from documentation).

NTC